discovery-provider/src/tasks/index_network_peers.py

```diff
@@ -15,15 +18,74 @@
 
 
 # Query the L1 set of audius protocol contracts and retrieve a list of peer endpoints
-def retrieve_peers_from_eth_contracts(self):
+def index_content_node_peers(self):
+    cid_metadata_client = update_network_peers.cid_metadata_client
     shared_config = update_network_peers.shared_config
     eth_web3 = update_network_peers.eth_web3
     redis = update_network_peers.redis
     eth_abi_values = update_network_peers.eth_abi_values
-    return fetch_all_registered_content_nodes(
+    content_nodes = fetch_all_registered_content_nodes(
         eth_web3, shared_config, redis, eth_abi_values
     )
 
+    content_peers = list(content_nodes)
+    # Update creator node url list in CID Metadata Client
+    # This list of known nodes is used to traverse and retrieve metadata from gateways
+    cid_metadata_client.update_cnode_urls(content_peers)
+    logger.info(f"index_network_peers.py | All known content peers {content_nodes}")
+
+
+def clique_propose(wallet: str, vote: bool):
```

Comment:
code recommendation -- have this fn return success / response and have logging be handled in the outer method. that way this one could be hypothetically unit tested cleanly.